pennylane/templates/decorator.py

```diff
@@ -54,10 +54,18 @@ def circuit():
     Returns:
         callable: The wrapper function
     """
+    # pylint: disable=import-outside-toplevel
 
     @wraps(func)
     def wrapper(*args, **kwargs):
-        with OperationRecorder() as rec:
+        import pennylane as qml
```

Comment:
hahaha I tried so many combinations (including having it at the top of the file), and constantly ran into cyclical import errors. This was the only combination that didn't result in a cyclical import error.

I'm okay with this for now, since

* we can remove it next week when we rip out the old core
* PennyLane has already been imported by this point, so there is no overhead here, it simply makes the namespace available to the function.